### PR TITLE
[CKEditor 5] Able to modify formula in read-only mode

### PR DESCRIPTION
**Pull request by carla-at-wiris**
## Description
It's possible to edit a formula by double clicking on it once you enable read-only mode in CKEditor5 demo.

## Steps to reproduce
1. Open ckeditor5 demo
2. Add formula to the editor
3. Enable read only mode by writing the following on the browser console: editor.isReadOnly = true
4. Double click on the formula and the modal window will open to edit the formula

## How it's been solved
By changing the double click handler function on the CKEditor5 mathtype package so that it takes into account the read only mode.

[#taskid 9796](https://wiris.kanbanize.com/ctrl_board/2/cards/9796/details/)

### CODE REVIEW

**Review by xaviripo — APPROVED** [review]

**Comment by dcanetma** [comment]
Fixes #158

File: packages/mathtype-ckeditor5/src/integration.js [hunk 1]
```diff
@@ -93,22 +93,24 @@ export default class CKEditor5Integration extends IntegrationModel {
...
-        if (typeof event.stopPropagation !== 'undefined') { // old I.E compatibility.
-          event.stopPropagation();
-        } else {
-          event.returnValue = false;
-        }
-        this.core.getCustomEditors().disable();
-        const customEditorAttr = element.getAttribute(Configuration.get('imageCustomEditorName'));
-        if (customEditorAttr) {
-          this.core.getCustomEditors().enable(customEditorAttr);
+    if (this.editorObject.isReadOnly == false) {
```
Comment (dcanetma) [inline packages/mathtype-ckeditor5/src/integration.js:96]: This line is not properly linted and it raises an error:

`96:38  error    Expected '===' and instead saw '=='    
`

Thanks to our awesome new command 'lint-js':

`npm run lint-js --route=packages/mathtype-ckeditor5`

**Review by dcanetma — CHANGES_REQUESTED** [review]
Please have a look to the comments since the code it may need to be tested.

File: packages/mathtype-ckeditor5/src/integration.js (hunk 1 above)
Comment (carlawiris) [reply packages/mathtype-ckeditor5/src/integration.js]: Done! Thank you

**Review by dcanetma — APPROVED** [review]

**Comment by dcanet-at-wiris** [comment]
Merged to stable from another branch.
Released on version v7.27.0.

**Pull request closed without merging**